app/src/test/java/com/example/benchproject/app/popular/movies/viewModel/PopularMoviesViewModelTest.kt

```diff
@@ -68,10 +68,10 @@ internal class PopularMoviesViewModelTest {
     @Test
     fun `when method get throwable from use case, errorSnackBar should run`() = runTest {
         coEvery { getPopularMoviesUseCase() } returns badResponse
-        every { popularMoviesFragmentNavigator.errorSnackBar(errorMessage) } just Runs
+        every { popularMoviesFragmentNavigator.errorSnackBar(errorMessage) { capture(slot) } } answers { slot.captured }
 
-        popularMoviesFragmentNavigator.errorSnackBar(errorMessage)
+        popularMoviesFragmentNavigator.errorSnackBar(errorMessage) { slot.captured }
```

Comment:
It's a view model test. why in "WHEN" section are you calling a navigator?

Reply:
I changed it